src/ex2/pokemonClient.js

```diff
@@ -0,0 +1,43 @@
+class PokemonClient {
+    constructor() {
+        this.API_BASE = `https://pokeapi.co/api/v2/`;
+    }
+
+
+    async catchPokemons(numbers) { // numbers -> array of numbers
+        const API = this.API_BASE + `pokemon/`;
+        const promises = [];
+        let res = [];
+        numbers.map((num) => {
+            promises.push(fetch(API + num + '/'));
+        })
```

Comment:
map actually returns a value, you could do something like that
```suggestion
        let res = [];
        const promises=numbers.map((num) => fetch(API + num + '/'));
        })
```